packages/keyring/src/pair/toJson.spec.ts

```diff
@@ -37,4 +38,58 @@ describe('toJson', (): void => {
       }
     });
   });
+
+  const ethereumPair = new Keyring({ type: 'ethereum' });
+  const pairOpts = {
+    meta: {
+      genesisHash: '0xc8d32cd0020b9da3804ef726f876604b7fcf5a799e687221a959f2dba5c78f18',
+      name: 'MyEthAccount',
+      tags: []
+    },
+    pairType: 'ethereum',
+    password: 'SomePassword',
+    suri: '0x728746aaa496d72abd7e3ef7bcdbaa87d5ba36ad612989ec782c0efb78c56866'
+  };
+
+  it('creates an unencoded output with no passphrase for ethereum account with ethereum type', (): void => {
+    const pair = ethereumPair.addFromUri(pairOpts.suri, pairOpts.meta, pairOpts.pairType as 'ethereum');
+
+    const json = pair.toJson();
+
+    expect(json).toMatchObject({
+      address: '0x5D0aa7d985DfF020174C907e3b57F5C8b5741013',
+      encoded: 'MFMCAQEwBQYDK2VwBCIEIHKHRqqkltcqvX4+97zbqofVujatYSmJ7HgsDvt4xWhmoSMDIQACqEZqFaw+I6dWa1sSwYCJn6wGo8VQ4/4GTV3EyCHIAiA=',
```

Comment:
Note that the fix from this PR focuses on the address, I'm actually not confident regarding the other fields.